static/js/main.js

```diff
@@ -375,6 +404,15 @@ function takeScreenshot(source, x, y, width, height, format = 'image/jpeg') {
     return canvas.toDataURL(format);
 }
 
+// Delete a screenshot, it is no longer displayed or saved.
```

Comment:
I'm not a fan of this comment because it is in a bit of an in-between space.
The purpose if pretty straightforward, so you could get away without a comment.
But since you have one and enumerate how the screenshot will be treated (not displayed or saved),
then you raise questions about what else is done with it.

Users may think: "Oh, there's a comment on this function and not the other functions, so there is probably a special situation. ... Humm... if it is not displayed or saved, what happens with the screenshot? It is still in the data but just not saved to the server?"